luthier/src/main/java/com/yahoo/bard/webservice/data/config/LogicalTableGroup.java

```diff
@@ -0,0 +1,15 @@
+// Copyright 2019 Oath Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.data.config;
+
+import com.yahoo.bard.webservice.table.LogicalTable;
+import com.yahoo.bard.webservice.table.TableIdentifier;
+
+import java.util.LinkedHashMap;
+
+/**
+ * Represents the set of LogicalTables that differs by granularity.
+ * This is used to build multiple LogicalTables from the config file.
+ */
+public class LogicalTableGroup extends LinkedHashMap<TableIdentifier, LogicalTable> {
```

Comment:
I thought we were going to make `LogicalTableGroup` an interface, and have the `LogicalTableDictionary` implement `LogicalTableGroup`?

Reply:
See above. I did not see this comment when writing that.